src/plugins/intel_cpu/tests/functional/single_layer_tests/instances/arm/convolution_backprop_data.cpp

```diff
@@ -0,0 +1,118 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "single_layer_tests/classes/convolution_backprop_data.hpp"
+#include "shared_test_classes/single_layer/convolution_backprop_data.hpp"
```

Comment:
use `"shared_test_classes/single_op/convolution_backprop_data.hpp"` instead, `single_layer` folder will be dropped in 2024.0

Reply:
@vurusovs corrected